spec/helpers/answers_helper_spec.rb

```diff
@@ -0,0 +1,132 @@
+require "spec_helper"
+
+RSpec.describe AnswersHelper, type: :helper do
+  describe "#answer_items" do
+    it "adds a query string to the link for each item" do
+      helper.answer_items.each do |item|
+        expect(item[:edit][:href]).to include("?change-answer")
```

Comment:
Thanks for adding a test for this - I should've done that when I made the change earlier.

Reply:
Actually, this should read thanks for adding all the tests that I should've done!